data_generation/CRF2013_2022.py

```diff
@@ -0,0 +1,807 @@
+    ncats["3.J.9"] = {"title": "NOx from Livestock"}
+
+    # Table 3.C
+    cats["3.C.1.b"]["children"] = [["3.C.1.b.i", "3.C.1.b.ii"]]
+    ncats["3.C.1.b.i"] = {"title": "Intermittently flooded Single aeration"}
+    ncats["3.C.1.b.ii"] = {"title": "Intermittently flooded Multiple aeration"}
+    cats["3.C.4"]["children"] = [["3.C.4.a"]]
+    ncats["3.C.4.a"] = {"title": "Other Non-Specified"}
+    # name for EST is "Non-specified"
+    # name for DEU is "Other"
+    # name for LVA is "other"
+    # name for CZE is "Other cultivation"
+
+    # Table 3.E
+    cats["3.E.1"]["children"] = [["3.E.1.a", "3.E.1.b", "3.E.1.c", "3.E.1.d"]]
+    ncats["3.E.1.a"] = {"title": "Savanna Woodland"}  # AUS
+    # name for SWE, CHE, CZE, HRV is "Forest land"
+    # name for MLT is "forest land"
+    ncats["3.E.1.b"] = {"title": "Savanna Grassland"}  # AUS
+    ncats["3.E.1.c"] = {"title": "Luxembourg"}  # LUX
+    ncats["3.E.1.d"] = {"title": "Other non-specified"}  # EST
+    # name for DNK, DKE, DNM is "All"
+    # name for DEU is "Unspecified"
+    # name for LVA is "Zone"
+    cats["3.E.2"]["children"] = [
+        ["3.E.2.a", "3.E.2.b", "3.E.2.c", "3.E.2.d", "3.E.2.e", "3.E.2.f", "3.E.2.g"]
+    ]
+    ncats["3.E.2.a"] = {"title": "Savanna Woodland"}  # AUS
+    ncats["3.E.2.b"] = {"title": "Savanna Grassland"}  # AUS
+    ncats["3.E.2.c"] = {"title": "Temperate Grassland"}  # AUS
+    ncats["3.E.2.d"] = {"title": "Grassland"}  # SWE, CHE, CZE, HRV
+    # name for MLT is "grassland"
+    ncats["3.E.2.e"] = {"title": "Luxembourg"}  # LUX
+    ncats["3.E.2.f"] = {"title": "Other non-specified"}  # EST
+    # name for DNK, DKE, DNM is "All
+    # name for DEU is "Unspecified"
+    # name for LVA is "Zone_"
+    ncats["3.E.2.g"] = {"title": "Tussock"}  # NZL
+
+    # Table 4
+
+    # entries under H. Other
+    cats["4.H"]["children"] = [[f"4.H.{i}" for i in range(1, 10)]]
+    # some of the subcategories refer to categories already included above
+    # but are used to report non-CO2 emissions
+    ncats["4.H.1"] = {"title": "Land converted to Settlement"}  # JPN
+    ncats["4.H.2"] = {"title": "Settlements Remaining Settlement"}  # USA, DEU, GBR
+    # name for DEU is "Settlements"
+    # name for GBR is "4.E Settlements"
+    ncats["4.H.3"] = {"title": "4.C Grassland"}  # GBR
+    ncats["4.H.4"] = {"title": "Biogenic NMVOCs from managed forest"}  # FRA
+    ncats["4.H.5"] = {"title": "Reservoir of Petit-Saut in French Guiana"}  # FRA
+    ncats["4.H.6"] = {"title": "N2O Emissions from Aquaculture Use"}  # AUS
+    ncats["4.H.7"] = {"title": "CH4 from artificial water bodies"}  # AUS
+    ncats["4.H.8"] = {"title": "Luxembourg"}  # LUX
+    ncats["4.H.9"] = {"title": "Other"}  # SWE, DEU
+    # name for SWE is "All Other"
+
+    # Table 5
+    cats["5.E"]["children"] = [
+        [
+            "5.E.1",
+            "5.E.2",
+            "5.E.3",
+            "5.E.4",
+            "5.E.5",
+            "5.E.6",
+            "5.E.7",
```

Comment:
```suggestion
            "5.E.7",
            "5.E.8",
            "5.E.9",
```

New in 2022 also needs to be added to children.